apps/vaporgui/MainForm.cpp

```diff
@@ -987,10 +957,6 @@ bool MainForm::eventFilter(QObject *obj, QEvent *event)
         _paramsWidgetDemo->Update(GetStateParams(), _paramsMgr);
 #endif
 
-        setUpdatesEnabled(false);
-        _controlExec->SyncWithParams();
-        setUpdatesEnabled(true);
```

Comment:
Please consider rebasing in the future as merging introduces unrelated changes which can make followup PR reviews more difficult.

Reply:
Agreed.  Sorry about that.

Reply:
No worries, thanks!